experimental/dds/tree2/src/test/snapshots/files/complete-3x3.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
note this isn't just a rename because I modified the snapshot verification/generation logic to only include the summary, not `ISummaryWithStats`.